src/views/tenants.js

```diff
@@ -0,0 +1,138 @@
+import React, { Component } from 'react';
+import BootstrapTable from 'react-bootstrap-table-next';
+import paginationFactory from 'react-bootstrap-table2-paginator';
```

Comment:
I'm getting an error trying to run this because 'react-bootstrap-table2-paginator' is missing from package.json, so we'll have to add it there first